### PR TITLE
PWN-4393 - New main toolbar with balance block

### PR DESCRIPTION
**Pull request by kuznetsov**
### Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-4393
https://p2pvalidator.atlassian.net/browse/PWN-4394

#### Toolbar with balance block redesign
- Balance loading elements were added to xml but weren't implemented
- ActionButtons were temporary hidden

### Screenshots

Screenshot 2022-08-04 at 20 11 02

### CODE REVIEW

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -9,69 +9,35 @@
...
 
-            <LinearLayout
+            <include
+                android:id="@+id/balance"
+                layout="@layout/layout_home_balance"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:background="@color/backgroundPrimary"
-                android:orientation="vertical">
+                android:layout_marginTop="80dp"
```
Comment (kabramovp2p) [inline app/src/main/res/layout/fragment_home.xml:32]: it is better to add 80dp as toolbarSize
and use it here and in height of layout_home_toolbar

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -104,11 +70,12 @@
         </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
         <org.p2p.wallet.common.ui.widget.ActionButtonsView
-            android:id="@+id/actionButtonsView"
+            android:id="@+id/action_buttons_view"
```
Comment (kabramovp2p) [inline app/src/main/res/layout/fragment_home.xml:73]: naming in new style needed actionButtonsViewHome - for example

File: app/src/main/java/org/p2p/wallet/restore/interactor/SecretKeyInteractor.kt [hunk 3]
```diff
@@ -88,7 +88,7 @@ class SecretKeyInteractor(
             putString(KEY_DERIVATION_PATH, path.stringValue)
         }
 
-        if (lookup) {
+        if (false) {
```
Comment (davran1001) [inline app/src/main/java/org/p2p/wallet/restore/interactor/SecretKeyInteractor.kt:91]: Forgot to revert

**Review by davran1001 — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/restore/interactor/SecretKeyInteractor.kt (hunk 3 above)
Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/restore/interactor/SecretKeyInteractor.kt:91]: Thanks!

**Review by kabramovp2p — APPROVED** [review]

**Pull request merged**